pdm_build.py

```diff
@@ -0,0 +1,35 @@
+import shutil
+import subprocess
+import sysconfig
+
+
+def pdm_build_hook_enabled(context):
+    # Disable hooks for sdist builds
+    return context.target != "sdist"
+
+
+def pdm_build_initialize(context):
+    context.ensure_build_dir()
+    # Inject compiled binary into scripts/, so it will be picked up to install
+    target = context.root / "scripts" / "bids-validator-deno"
```

Comment:
are you sure about this one?  on one hand it is reflecting pypi package name, but on the other -- it is "exactly" the same what you get when you install with deno but that one gives you `bids-validator`.  If there would be some `bids-validator` which we would install from pypi eventually --  how underlying tools would need to decide on what CLI "interface" to use to talk to `bids-validator` ?

Reply:
It seems like a bad idea to me for both the `bids-validator` and `bids-validator-deno` PyPI packages to both install executables to the same name. I think we'd be asking for bug reports.

Reply:
agree. But I also think that we need to keep executable named the same way regardless how it is installed. I feel like we better rename the much less used ATM `bids-validator` from python package (I even forgot that it exists) into e.g. `bids-validator-py` or `py-bids-validator` or something along the lines.

Reply:
IMO a Python package should have precedence on PyPI.

Possibly I'm misunderstanding something in all this.